packages/react-composites/src/composites/MeetingComposite/MeetingComposite.tsx

```diff
@@ -43,6 +43,30 @@ export interface MeetingCompositeProps extends BaseCompositeProps<CallCompositeI
    * URL that can be used to copy a meeting invite to the Users clipboard.
    */
   meetingInvitationURL?: string;
+  /**
+   * Call control options to change what buttons show on the meeting composite control bar.
+   */
+  meetingCallControlOptions?: MeetingCallControlOptions;
```

Comment:
Can we also allow for completely hiding the control bar to match the call composite callcontrols interface:
![image](https://user-images.githubusercontent.com/2684369/146577931-a7897d11-705d-456b-b952-645106281fe4.png)

(though this is a good example of a change that would be better as a separate PR as although its similar, is a distinctly different feature)

Reply:
Will make separate PR for this issue.